src/3b_count_aca.jl

```diff
@@ -0,0 +1,130 @@
+"""
+Collate number of potentially suitable locations per reef, as defined by
+GBRMPA Features.
+"""
+
+using Rasters
+import GeoDataFrames as GDF
+import ArchGDAL as AG
+using CSV
+
+using Statistics, StatsBase
+using Glob
+
+include("common.jl")
+
+reef_features = GDF.read(REEF_PATH)
+reef_features.region .= ""
+reef_features.reef_name .= ""
+reef_features.n_flat .= 0
+reef_features.n_slope .= 0
+reef_features.flat_ha .= 0.0
+reef_features.slope_ha .= 0.0
+
+reef_features.flat_scr .= 0.0
+reef_features.slope_scr .= 0.0
+
+"""
+    count_suitable(raster, reef)::Int64
+
+Count number of suitable pixels in an area as defined by the `reef` geometry, for the
+given `raster`.
+
+This has to be applied one by one for each reef as any malformed geometries lead to a crash.
+"""
+function count_suitable(raster, reef)::Union{Int64, Missing}
```

Comment:
Likely not anything to do with this set of changes, but please update the function signature in the docstring to match implementation

```suggestion
    count_suitable(raster, reef)::Union{Int64, Missing}

Count number of suitable pixels in an area as defined by the `reef` geometry, for the
given `raster`.

This has to be applied one by one for each reef as any malformed geometries lead to a crash.
"""
function count_suitable(raster, reef)::Union{Int64, Missing}
```